src/scf/newton.jl

```diff
@@ -158,10 +158,10 @@ function solve_ΩplusK(basis::PlaneWaveBasis{T}, ψ, rhs, occupation;
     J = LinearMap{T}(ΩpK, size(rhs_pack, 1))
 
     # solve (Ω+K) δψ = rhs on the tangent space with CG
-    δψ = cg(J, rhs_pack, Pl=FunctionPreconditioner(f_ldiv!),
-            reltol=0, abstol=tol_cg, verbose=verbose)
+    δψ, history = cg(J, rhs_pack, Pl=FunctionPreconditioner(f_ldiv!),
+                  reltol=0, abstol=tol_cg, verbose=verbose, log=true)
 
-    unpack(δψ)
+    (; δψ=unpack(δψ), history)
```

Comment:
I quite like this pattern of returning named tuples for non-trivial functions: it's sufficiently flexible that we can return whatever extra information we want, and the caller is free to ignore